ui/ui-expenses/src/screens/prices/prices.reactions.js

```diff
@@ -0,0 +1,27 @@
+export const add = ((action, store, { http }) => {
+  http('EXPENSES')
+    .post(
+      '/api/expenses',
+      {
+        client: 'CLIENT',
+        price: store.ui.price.get(),
+        fileId: store.data.fileId.get(),
+        user: store.data.profile.get().name,
+        needRefund: true,
+      },
+      { credentials: 'include' },
+    )
+})
+
+export const setPrice = (({ payload }, store) => {
+  store.ui.price.set(payload)
+})
+
+export const response = ((action, store) => {
+  store.ui.response.set('price adding 👏')
```

Comment:
`price added` is better ! But inconsistent with the rest of the app which is in french 😅